kv/dist_sender.go

```diff
@@ -402,7 +402,7 @@ func (ds *DistSender) sendRPC(desc *proto.RangeDescriptor,
 	}
 
 	// Copy and rearrange the replicas suitably, then return the desired order.
-	replicas := proto.ReplicaSlice(append([]proto.Replica(nil), desc.Replicas...))
+	replicas := append(proto.ReplicaSlice{}, desc.Replicas...)
```

Comment:
Understood. Does `(append(proto.ReplicaSlice(nil), desc.Replicas...)` differ from `append(proto.ReplicaSlice{}, desc.Replicas...)` in the total number of allocations?